seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json

```diff
@@ -165,10 +165,10 @@
       ],
       "solutions": [],
       "hints": [],
-      "type": "basejump",
+      "type": "waypoint",
```

Comment:
@JacksonBates I've QA'd everything and everything looks great except for this. If you change this to "backend" like @systimotic recommends, this will render properly, too.